runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineOptions.java

```diff
@@ -82,6 +82,13 @@
   void setUpdate(boolean value);
 
   /**
+   * Where the runner should generate a template file.
+   */
+  @Description("Where the runner should generate a template file.")
```

Comment:
We will soon have more than just GCS and local files. Explicitly check that the path is one of these two?